src/Annotations/AppKernel.php

```diff
@@ -103,6 +102,15 @@ public function __construct(string $environment = 'dev')
                 'authority' => '',
                 'group' => '',
             ],
+            'common_mark.environment' => ($config['common_mark']['environment'] ?? []) + [
+                'allow_unsafe_links' => false,
+            ],
+            'html_purifier' => ($config['html_purifier'] ?? []) + [
+                'Cache.SerializerPath' => __DIR__.'/../../var/cache/html_purifier',
+            ],
+            'html_converter' => ($config['html_converter'] ?? []) + [
```

Comment:
I think this is a leftover to be removed?

Reply:
Will remove